perf/perf_hv_gpci.py

```diff
@@ -124,5 +137,6 @@ def test_gpci_events(self):
             self.gpci_events(event)
 
     def tearDown(self):
+        process.run('userdel -f test', sudo=True, ignore_status=True)
```

Comment:
please have if user present

Reply:
@PraveenPenguin have addressed all review comments and updated, please check.